sentence_transformers/models/CLIPModel.py

```diff
@@ -72,7 +73,11 @@ def tokenize(self, texts, padding: Union[str, bool] = True):
             encoding["pixel_values"] = image_features.pixel_values
 
         encoding["image_text_info"] = image_text_info
-        return encoding
+        device = get_device_name()
+        if device == "hpu":
+            return dict(encoding)
+        else:
+            return encoding
```

Comment:
```suggestion
        return dict(encoding)
```
I think it should be fine to just return a dictionary here regardless, rather than the BatchEncoding instance.